packages/mobile/ios/Podfile

```diff
@@ -49,7 +49,6 @@ target 'AudiusReactNative' do
       # necessary for Mac Catalyst builds
       :mac_catalyst_enabled => false
     )
-    __apply_Xcode_12_5_M1_post_install_workaround(installer)
```

Comment:
https://react-native-community.github.io/upgrade-helper/?from=0.66.5&to=0.70.6 it seems like it only applies to older Xcode, do we really want to remove?

Reply:
won't pod install exclude arm64 after you run it?

Reply:
I don't think so - that was a different couple lines that did it manually